repo/migrations/Migration009_test.go

```diff
@@ -0,0 +1,377 @@
+	}
+	for keysRows.Next() {
+		var coin, scriptAddres, key string
+		var purpose, keyIndex, used int
+		err := keysRows.Scan(&coin, &scriptAddres, &purpose, &keyIndex, &used, &key)
+		if err != nil {
+			t.Error(err)
+		}
+		if coin != "BCH" {
+			t.Error("Failed to return correct coin type")
+		}
+		if scriptAddres != "key1" {
+			t.Error("Failed to return correct scriptAddress")
+		}
+		if key != "keyhex" {
+			t.Error("Failed to return correct key")
+		}
+		if purpose != 0 {
+			t.Error("Failed to return correct purpose")
+		}
+		if used != 0 {
+			t.Error("Failed to return correct used")
+		}
+		if keyIndex != 5 {
+			t.Error("Failed to return correct keyIndex")
+		}
+	}
+
+	// Verify utxos data
+	utxosRows, err := db.Query(selectUtxosSQL, "abc:0")
+	if err != nil {
+		t.Fatal(err)
+	}
+	coinColumnExists = false
+	columns, err = utxosRows.ColumnTypes()
+	if err != nil {
+		t.Fatal(err)
+	}
+	for _, c := range columns {
+		if c.Name() == "coin" {
+			coinColumnExists = true
+		}
+	}
+	if coinColumnExists == false {
+		t.Error("Expected coin column to exist on utxos")
+	}
+	for utxosRows.Next() {
+		var coin, outpoint, scriptPubkey string
+		var value, height, watchOnly int
+		err := utxosRows.Scan(&coin, &outpoint, &value, &height, &scriptPubkey, &watchOnly)
+		if err != nil {
+			t.Error(err)
+		}
+		if coin != "BCH" {
+			t.Error("Failed to return correct coin type")
+		}
+		if outpoint != "abc:0" {
+			t.Error("Failed to return correct outpoint")
+		}
+		if value != 3000 {
+			t.Error("Failed to return correct value")
+		}
+		if height != 150000 {
+			t.Error("Failed to return correct height")
+		}
+		if watchOnly != 0 {
+			t.Error("Failed to return correct watchOnly")
+		}
+	}
+
+	// Verify stxos data
+	stxosRows, err := db.Query(selectStxosSQL, "abc:0")
+	if err != nil {
+		t.Fatal(err)
+	}
+	coinColumnExists = false
+	columns, err = stxosRows.ColumnTypes()
+	if err != nil {
+		t.Fatal(err)
+	}
+	for _, c := range columns {
+		if c.Name() == "coin" {
+			coinColumnExists = true
+		}
+	}
+	if coinColumnExists == false {
+		t.Error("Expected coin column to exist on stxos")
+	}
+	for stxosRows.Next() {
+		var coin, outpoint, scriptPubkey, spendTxid string
+		var value, height, watchOnly, spendHeight int
+		err := stxosRows.Scan(&coin, &outpoint, &value, &height, &scriptPubkey, &watchOnly, &spendHeight, &spendTxid)
+		if err != nil {
+			t.Error(err)
+		}
+		if coin != "BCH" {
+			t.Error("Failed to return correct coin type")
+		}
+		if outpoint != "abc:0" {
+			t.Error("Failed to return correct outpoint")
+		}
+		if value != 3000 {
+			t.Error("Failed to return correct value")
+		}
+		if height != 150000 {
+			t.Error("Failed to return correct height")
+		}
+		if watchOnly != 0 {
+			t.Error("Failed to return correct watchOnly")
+		}
+		if spendHeight != 150001 {
+			t.Error("Failed to return correct spendHeight")
+		}
+		if spendTxid != "1234" {
+			t.Error("Failed to return correct spendTxid")
+		}
+	}
```

Comment:
Testing pattern to adopt: Memoize the asserted values to indicate the actual value doesn't matter and only needs to match the beginning, improves maintainability, and the variable names can be used to indicate what type of value it is suppose to be.